src/Client/OfficeInterop/OfficeInterop.fs

```diff
@@ -1957,12 +1954,128 @@ let validateAnnotationTable () =
                         [InteropLogging.Msg.create InteropLogging.Warning $"Table is not a valid ARC table / ISA table: {ex.Message}. The column {header} is not valid! It needs further inspection what causes the error.";
                         ])
                 else
-                    [InteropLogging.Msg.create InteropLogging.Warning $"The annotation table {excelTable.name} is valid"]
+                    []
 
             return messages
         }
     )
 
+let fillEmptyBuildingBlocks () =
```

Comment:
rename to `autoCompleteFromExistingTerms`, add extensive comment in xml style to explain functionality.

Reply:
fixed